packages/sdk-cli/src/commands/connect.ts

```diff
@@ -71,14 +79,104 @@ export const connectAction = async (
   return true;
 };
 
+export async function startLocalRelayAction(cli: vorpal) {
+  if (!(await installRelayPkgPrompt(cli))) {
+    return;
+  }
+
+  const relayJsPath = await relayEntryPointPath();
+  // FORK:
+  // https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options
+  // Unlike POSIX fork(), child_process.fork() creates a completely separate V8 process with its own memory.
+  // Dangers of POSIX fork() (https://www.evanjones.ca/fork-is-dangerous.html) don't apply.
+  child_process.fork(relayJsPath, {
+    detached: true,
+    // We don't want to read parent's stdin from child process, but we want to share the same stdout/stderr.
```

Comment:
What happens once the shell is closed and that stdin/stderr no longer exists? Piping stdout to a log file may be more appropriate.